src/ResourceManager/Sql/Commands.Sql.Test/packages.config

```diff
@@ -2,13 +2,14 @@
 <packages>
   <package id="Hyak.Common" version="1.0.3" targetFramework="net45" />
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.4.1-preview" targetFramework="net45" />
```

Comment:
Common authentication was moved to powershell repo. Please reference it from there